Tests/test_papersizes.py

```diff
@@ -0,0 +1,30 @@
+from PyPDF2 import papersizes
+import pytest
+
+
+def test_din_a0():
+    dim = papersizes.PaperSize.A0
+    area_square_pixels = float(dim.width) * dim.height
```

Comment:
In case anybody wonders about the weird float conversion: https://twitter.com/_martinthoma/status/1517767880686936065